---
source: src/validation/tests/generic_validation_tests.rs
expression: make_readable(&diagnostics)
---
SyntaxError { message: "Invalid assignment: cannot assign 'DATE_AND_TIME' to 'CHAR'", range: [SourceRange { range: 136..137 }], err_no: var__invalid_assignment }
SyntaxError { message: "Invalid type nature for generic argument. DATE_AND_TIME is no Char.", range: [SourceRange { range: 136..137 }], err_no: type__invalid_nature }
SyntaxError { message: "Invalid assignment: cannot assign 'DATE_AND_TIME' to 'CHAR'", range: [SourceRange { range: 203..204 }], err_no: var__invalid_assignment }
SyntaxError { message: "Invalid type nature for generic argument. DATE_AND_TIME is no Char.", range: [SourceRange { range: 203..204 }], err_no: type__invalid_nature }
SyntaxError { message: "Invalid assignment: cannot assign 'DATE' to 'CHAR'", range: [SourceRange { range: 271..272 }], err_no: var__invalid_assignment }
SyntaxError { message: "Invalid type nature for generic argument. DATE is no Char.", range: [SourceRange { range: 271..272 }], err_no: type__invalid_nature }
SyntaxError { message: "Invalid assignment: cannot assign 'TIME_OF_DAY' to 'CHAR'", range: [SourceRange { range: 338..339 }], err_no: var__invalid_assignment }
SyntaxError { message: "Invalid type nature for generic argument. TIME_OF_DAY is no Char.", range: [SourceRange { range: 338..339 }], err_no: type__invalid_nature }
SyntaxError { message: "Invalid assignment: cannot assign 'TIME_OF_DAY' to 'CHAR'", range: [SourceRange { range: 406..407 }], err_no: var__invalid_assignment }
SyntaxError { message: "Invalid type nature for generic argument. TIME_OF_DAY is no Char.", range: [SourceRange { range: 406..407 }], err_no: type__invalid_nature }

